lms/static/sass/shared/_footer-edx.scss

```diff
@@ -19,20 +19,72 @@ footer#footer-edx-v3 {
   padding: $baseline*0.75 0;
   border-top: 1px solid $courseware-button-border-color;
 
+  .column-1 {
+    flex: 0 0 100%;
+    max-width: 100%;
+  }
+
+  .column-styles {
+    position: relative;
+    width: 100%;
+    min-height: 1px;
+    padding-right: 15px;
+    padding-left: 15px;
+  }
+
+  .column-2 {
+    flex: 0 0 100%;
+    max-width: 100%;
+  }
+
+  .column-3 {
+    flex: 0 0 50%;
+    max-width: 50%;
+  }
+
+  .column-2-1 {
+    flex: 0 0 100%;
+    max-width: 100%;
+  }
+
+  .column-2-2 {
+    flex: 0 0 100%;
+    max-width: 100%;
+  }
+
+  .column-3 {
+    flex: 0 0 100%;
+    max-width: 100%;
+  }
+
+  .column-3-1 {
+    flex: 0 0 100%;
+    max-width: 100%;
+  }
+
+  .column-3-2 {
+    flex: 0 0 100%;
+    max-width: 100%;
+  }
+
+  a.social-links {
+    font-size: 35px;
+  }
+
   .first-div-border {
-    @include border-left(1px solid $courseware-button-border-color);
+    @include border-left(none);
   }
 
   .second-div-border {
-    @include border-left(1px solid $courseware-button-border-color);
+    @include border-left(none);
   }
 
   .third-div-border {
-    @include border-left(1px solid $courseware-button-border-color);
+    @include border-left(none);
   }
 
   .fourth-div-border {
-    @include border-left(1px solid $courseware-button-border-color);
+    @include border-left(none);
   }
```

Comment:
More repetition. Either use a more generic class, or comma-separate the selectors so you only have to declare  border-left once.

Reply:
Actually our borders changes for each design if you that is why we have to do that. for each screen.

Reply:
@hamzamunir7300 it's more that you are setting the same value for all four borders at the same time. So you could update this to;
```
.first-div-border,
.second-div-border,
.third-div-border,
.fourth-div-border {
  @include border-left(none);
}
```